oio/api/object_storage.py

```diff
@@ -456,7 +456,7 @@ def container_snapshot(self, account, container, dst_account,
                 fullpath = encode_fullpath(
                     dst_account, dst_container, obj['name'], obj['version'],
                     obj['content'])
-                storage_method = STORAGE_METHODS.load(obj['chunk_method'])
+                storage_method = STORAGE_METHODS.load(obj['chunk-method'])
```

Comment:
Other parts use `chunk_method` (for example https://github.com/open-io/oio-sds/blob/309db2cedf8093aaa710caf5aca84559c7ad86a7/oio/api/object_storage.py#L1110), it seems confusing to mix both of them